test/functional/cmdLineTests/criu/criu_nonPortable.xml

```diff
@@ -1061,4 +1065,80 @@
     <output type="failure" caseSensitive="yes" regex="no">Could not dump the JVM processes, err=-70</output>
     <output type="failure" caseSensitive="yes" regex="no">User requested Java dump using</output>
   </test>
+
+  <test id="Test Ghost File Limit -- negative">
+    <command>bash $SCRIPPATH$ $TEST_RESROOT$ $JAVA_COMMAND$ "$JVM_OPTIONS$ --add-exports java.base/openj9.internal.criu=ALL-UNNAMED" $MAINCLASS_GHOSTFILE_TEST$ negative 1 false false</command>
```

Comment:
Suggest to define a variable for `--add-exports java.base/openj9.internal.criu=ALL-UNNAMED`.